src/Bindings/Plugin.h

```diff
@@ -69,6 +69,7 @@ class cPlugin
 	virtual bool OnKilled                   (cEntity & a_Victim, TakeDamageInfo & a_TDI, AString & a_DeathMessage) = 0;
 	virtual bool OnKilling                  (cEntity & a_Victim, cEntity * a_Killer, TakeDamageInfo & a_TDI) = 0;
 	virtual bool OnLogin                    (cClientHandle & a_Client, UInt32 a_ProtocolVersion, const AString & a_Username) = 0;
+	virtual bool OnLoginForge               (cClientHandle & a_Client) = 0;
```

Comment:
You need to pass `const AStringMap &`. Unfortunately C++'s param matching inherits the ugly custom from C that almost anything can be converted to a `bool`